test/e2e/fixtures/fixtures.go

```diff
@@ -73,3 +73,42 @@ var NginxPluginDefinition = &greenhousev1alpha1.PluginDefinition{
 		},
 	},
 }
+
+var TestHookPluginDefinition = &greenhousev1alpha1.PluginDefinition{
+	TypeMeta: metav1.TypeMeta{
+		Kind:       "PluginDefinition",
+		APIVersion: greenhousev1alpha1.GroupVersion.String(),
+	},
+	ObjectMeta: metav1.ObjectMeta{
+		Name:      "test-hooks",
+		Namespace: test.TestNamespace,
+	},
+	Spec: greenhousev1alpha1.PluginDefinitionSpec{
+		Version:        "0.1.0",
+		Weight:         ptr[int32](0),
+		DisplayName:    "Test Hooks",
+		Description:    "Test Hooks",
+		DocMarkDownUrl: "https://raw.githubusercontent.com/cloudoperators/greenhouse-extensions/main/alerts/README.md",
+		Icon:           "https://raw.githubusercontent.com/cloudoperators/greenhouse-extensions/main/alerts/logo.png",
+		HelmChart: &greenhousev1alpha1.HelmChartReference{
+			Name:       "./../../pkg/test/fixtures/testHook",
+			Repository: "dummy",
+			Version:    "0.1.0",
+		},
+		UIApplication: &greenhousev1alpha1.UIApplicationReference{
+			Name:    "supernova",
+			Version: "latest",
+		},
+		Options: []greenhousev1alpha1.PluginOption{
+			{
+				Name:    "hook_enabled",
+				Type:    "bool",
+				Default: &apiextensionsv1.JSON{Raw: []byte("false")},
+			},
+		},
+	},
+}
+
+func ptr[T any](value T) *T {
+	return &value
+}
```

Comment:
the `Weight` field is not required and does not make a difference for the controllers. This field is just used to order the PluginDefinitions in the UI. If you can leave out some of the other optional fields as well, this test can also use the helpers from https://github.com/cloudoperators/greenhouse/blob/main/pkg/test/resources.go#L164 

For the PluginDefinition the only thing missing afaics is the function to set the PluginOptions.

Since the rest of the tests already use the `test.TestSetup(..)` it would be nice to use them for the new test fixtures as well.

Reply:
Done